scripts/tracevis.py

```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Did you write this? I think this script is missing an author (although I guess we can always resort to a `git blame`).

Reply:
As far as I know, the original version is from [SalvatoreDiGirolamo](https://github.com/SalvatoreDiGirolamo/tracevis). Noah adapted it to a Python script for the Snitch trace format and I continued development for MemPool.

I will describe this in a comment.